akka-docs/src/main/paradox/stream/stream-graphs.md

```diff
@@ -108,6 +108,12 @@ Let's imagine we want to provide users with a specialized element that given 3 i
 the greatest int value of each zipped triple. We'll want to expose 3 input ports (unconnected sources) and one output port
 (unconnected sink).
 
+@@@ note
+
+The ZipWith has `ZipWith[In0, In1, Out](f: (In0, In1) => Out)` signature.
```

Comment:
Thanks! How about improving it a bit (the signature of ZipWith is a bit of a lie here, it is multiple classes internally actually)

```
While the above example shows composing two 2-input `ZipWith`s, in reality ZipWIth already provides numerous overloads including a 3 (and many more) parameter versions. So this could be implemented as `ZipWith((a, b, c) => out)`.
```